packages/web-app-files/src/components/AppBar/AppBar.vue

```diff
@@ -32,13 +32,18 @@
         </div>
       </div>
       <div class="files-app-bar-actions">
-        <div class="oc-flex-1 oc-flex oc-flex-start" style="gap: 15px">
+        <div class="oc-flex-1 oc-flex oc-flex-start">
           <slot v-if="showActionsOnSelection || selectedFiles.length === 0" name="actions" />
-          <size-info
+          <oc-button
```

Comment:
We could easily implement the `Clear selection` action as a file action (see actions mixins). That would remove the need for special code here. What do you think?

Reply:
Good idea, I'll do it that way 👍